src/styles/override.css

```diff
@@ -85,6 +85,9 @@ body {
     margin-left: 20px;
     margin-top: 10px;
 }
+.css404{
+    margin-top: 50px;
```

Comment:
I'm being fussy here, but you'd be best off using Bootstrap's inbuilt spacing to set these margins.

Try that first, and see if you can achieve the same result.

https://getbootstrap.com/docs/4.0/utilities/spacing/

This applies to anywhere you've set margins manually.